tvb_epilepsy/tvb_api/simulator_tvb.py

```diff
@@ -142,38 +162,54 @@ def prepare_for_h5(self, settings):
 
         return epileptor_model_h5_model
 
+    def configure_model(self, hypothesis=None, **kwargs):
+
+        if hypothesis is None:
+            hypothesis = self.hypothesis
+
+        self.model = model_build_dict[self.model._ui_name](hypothesis, **kwargs)
+
+    def conifigure_initial_conditions(self, initial_conditions=None):
+
+        if isinstance(initial_conditions, numpy.ndarray):
+            self.simTVB.initial_conditions = initial_conditions
+
+        else:
+            self.simTVB.initial_conditions = self.prepare_initial_conditions(self.hypothesis,
+                                                                             self.simTVB.good_history_shape[0])
 
 ###
 # A helper function to make good choices for simulation settings, noise and monitors
 ###
 
-def setup_simulation(model_name, hypothesis, dt, sim_length, monitor_period, zmode=numpy.array("lin"), scale_time=1,
-                     noise_instance=None, noise_intensity=None,
+def setup_simulation(hypothesis, head_connectivity, dt, sim_length, monitor_period, model_name="EpileptorDP",
+                     zmode=numpy.array("lin"), scale_time=1, noise_instance=None, noise_intensity=None,
```

Comment:
It's a bug-waiting-to-happen to use a mutable data structure as a keyword default; better to use `None` and, below, `self.zmode = zmode or numpy.array("lin")`.